packages/assets-controllers/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@metamask/assets-controllers",
+  "version": "0.0.0",
+  "description": "Controllers which manage interactions involving ERC-20, ERC-721, and ERC-1155 tokens (including NFTs)",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/controllers/tree/main/packages/assets-controllers#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/controllers/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/controllers.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "changelog:validate": "../../scripts/validate-changelog.sh",
+    "doc": "../../scripts/doc.sh",
+    "test": "../../scripts/test.sh",
+    "test:watch": "../../scripts/test-watch.sh"
+  },
+  "dependencies": {
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/contracts": "^5.7.0",
+    "@ethersproject/providers": "^5.7.0",
+    "@metamask/base-controller": "workspace:~",
+    "@metamask/contract-metadata": "^1.35.0",
+    "@metamask/controller-utils": "workspace:~",
+    "@metamask/metamask-eth-abis": "3.0.0",
+    "@metamask/network-controller": "workspace:~",
+    "@metamask/preferences-controller": "workspace:~",
+    "@types/uuid": "^8.3.4",
+    "@types/web3": "^1.0.6",
+    "abort-controller": "^3.0.0",
+    "async-mutex": "^0.2.6",
+    "eth-query": "^2.1.2",
+    "eth-rpc-errors": "^4.0.0",
+    "ethereumjs-util": "^7.1.4",
```

Comment:
Previously this was
```suggestion
    "ethereumjs-util": "^7.0.10",
```

Reply:
Fixed here: https://github.com/MetaMask/controllers/pull/831/commits/b23a926a568ee825a72c75bd3ef40fc0b8156106